### DISCUSSION TITLE
Welcome to front-end-interview-handbook Discussions!

### DISCUSSION THREAD

**Discussion by yangshun** [yangshun/front-end-interview-handbook#336] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tapanchudasama** [comment]
Hello folks.

Been using the greatfrontend platform as a lifetime subscriber, and have only good things to say.

It'd be even better if there was a feature to bookmark questions and have them for quick reference.

Thanks.
